### PR TITLE
Updated unstyled list guidance to cover lst-none

### PR DESCRIPTION
**Pull request by pjackson28**

### CODE REVIEW

**Comment by pjackson28** [comment]
It might be a good idea to clean up the commits. I don't have access to a proper dev workstation to do that (had to use the Web interface).

**Comment by duboisp** [comment]
Thanks @pjackson28

Related to https://github.com/wet-boew/wet-boew/pull/8008

File: site/pages/v4/design/lists-en.hbs [hunk 1]
```diff
@@ -217,9 +226,19 @@
       </div>
       <div class="col-md-4">
 	  <h5 class="mrgn-tp-0">Code</h5>
-        <pre><code>&lt;ul <strong>class=&quot;list-unstyled lst-spcd&quot;</strong>&gt;
+      <p>Unstyled list:</p>
+        <pre><code>&lt;ul <strong>class=&quot;list-unstyled&quot;</strong>&gt;
   &lt;li&gt;...&lt;/li&gt;
 &lt;/ul&gt;</code></pre>
+      <p>Unstyled nested list:</p>
+        <pre><code>&ltul&gt;
```
Comment (nschonni) [inline site/pages/v4/design/lists-en.hbs]: Missing semi for `&lt;`

Comment (pjackson28) [inline site/pages/v4/design/lists-en.hbs]: Added the missing semi colon but I don't have the ability to squash the commits as I don't have access to an appropriate dev machine and don't know how to do it through the Web interface.

**Comment by nschonni** [comment]
@pjackson28 there is a "Squash and Merge" option in the web UI's green button. Can't do it as you go, but you do get a clean merge in the end

**Comment by pjackson28** [comment]
@nschonni Good to know! Will someone else be doing the squash and merge or should I do it myself?

**Comment by nschonni** [comment]
I'll let you have your first try at the new merge button :wink:

**Pull request merged**